src/lib/constants/direction-properties.ts

```diff
@@ -1,58 +1,10 @@
-/* eslint-disable import/no-cycle */
-import { IDirection } from '../types';
+import { useSelector } from 'react-redux';
+import { RootStateType } from '../../store/rootReducer';
 
-export type DirectionPropertiesType = {
-  [key: string]: IDirection;
-};
+export const DirectionProperties = () => {
```

Comment:
we don't need this file anymore, don't we?

Reply:
yes, I just forgot to delete. Thank you!